src/components/title.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import * as fonts from '../assets/fonts'
+
+type TitleSize = "small" | "medium" | "large" | "xlarge" | "xxlarge"
+
+interface TitleProps extends React.HTMLProps<JSX.Element> {
+  titleSize?: TitleSize
+}
+
+const titleSizes = {
+  small: '25px',
+  medium: '30px',
+  large: '37px',
+  xlarge: '50px',
+  xxlarge: '72px'
+}
+
+const Title: React.SFC<TitleProps> = props => (
+  <div className={props.className}>
```

Comment:
@yuki24 oh yeah. what @alloy said. sorry, missed the "Native" part.

There's a project called [React Native Web](https://github.com/necolas/react-native-web) that brings react native components to the web but otherwise, React and React Native use different tags.